### ISSUE TITLE
Unable to build docker image due firefox dependency

### ISSUE THREAD

**Issue by wazowsk1** [ppodgorsek/docker-robot-framework#222]
When I try to build the container from the master branch, I get the following error at the install firefox step.

```
ERROR: unsatisfiable constraints:
  firefox (missing):
    required by: world[firefox~70.0]
```

when using the following command
```docker build  .```

**Comment by UltimateDogg** [comment]
not sure what happened, but i see in the apk repo there is only firefox-esr 68 now. Not sure where it went, it was there pretty recently.

**Comment by ppodgorsek** [comment]
Fixed in PR #219
